netci.groovy

```diff
@@ -55,7 +55,7 @@ def targetGroupOsMapInnerloop = ['netcoreapp': ['Windows_NT', 'Ubuntu14.04', 'Ub
         def isLocal = (localType == 'local')
 
         def newJobName = 'code_coverage_windows'
-        def batchCommand = 'call build.cmd && call build-tests.cmd -coverage -outerloop -- /p:IsCIBuild=true'
+        def batchCommand = 'call build.cmd && call build-tests.cmd -coverage -outerloop -- /p:IsCIBuild=true /p:CodeCoverageAssemblies="System.Private.CoreLib"'
```

Comment:
This adds System.Private.CoreLib rather than replacing the target assemblies, right? i.e. we won't get coverage data only for corelib as a result of this change?

Reply:
Just validated for myself that it correctly adds rather than replaces.